app.js

```diff
@@ -0,0 +1,8 @@
+var portfolioArray = [];
```

Comment:
So good effort here! But this is instantiating an object and not a constructor function, take a closer look at what should this look like. Hint: look at the lab from tonight